include/pgduckdb/pgduckdb_planner.hpp

```diff
@@ -2,13 +2,11 @@
 
 #include "duckdb.hpp"
 
-extern "C" {
-#include "postgres.h"
-#include "optimizer/planner.h"
-}
-
+#include "pgduckdb/pg/declarations.hpp"
 #include "pgduckdb/pgduckdb_duckdb.hpp"
 
+#include "pgduckdb/utility/cpp_only_file.hpp" // Must be last include.
```

Comment:
Do we really want to include this in headers? That means that you cannot include it after postgres includes in a `cpp` file I guess. Do we really want that?

Reply:
I think it is quite helpful to make sure we don't include PG headers in the current header file and indeed because it is a header it must be `#included` before the PG header inclusion in cpp files.

I find it quite useful to iterate for now, but I'm happy to remove it from all headers when our code is in a stable place.

Reply:
Okay makes sense.